modules/coremark/Kconfig

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig COREMARK
+	bool "CoreMark benchmark"
+
+if COREMARK
+
+config COREMARK_ITERATIONS
+	int "Number of iterations to run"
+	range 10 10000
+	default 1000
+	help
+	  Specify the number of iterations to run the benchmark.
+	  CoreMark should execute for at least 10 seconds to have valid results.
+	  The minimum number of iterations depends on the platform and its configuration.
+
+config COREMARK_THREADS_NUMBER
+	int "Number of threads for the parallel benchmark execution"
+	default 1
```

Comment:
Since you removed range, should we add a build assert in code to ensure it's a positive number?